cvat/schema.yml

```diff
@@ -8888,6 +9040,31 @@ components:
         * `natural` - NATURAL
         * `predefined` - PREDEFINED
         * `random` - RANDOM
+    StatisticsItem:
```

Comment:
I recommend adding some help info for `granularity`, `default_view`, `dataseries` and `transformations` fields to clarify about their contents for API users.